src/main/kotlin/lotto/model/Constants.kt

```diff
@@ -0,0 +1,32 @@
+package lotto.model
+
+const val error = "[ERROR]"
+enum class ERROR(val message: String) {
+    COMMA_REQUIRED("$error 쉼표를 포함해주세요"),
+    NOT_NUMBER("$error 숫자를 입력해주세요"),
+    OUT_OF_RANGE("$error 1부터 45까지의 숫자만 입력이 가능합니다"),
+    DUPLICATED_NUMBER("$error 중복된 숫자는 입력할 수 없습니다."),
+    INVALID_SIZE("$error 6개의 숫자만 입력이 가능합니다."),
+    INVALID_PURCHASE_AMOUNT("$error 구입 금액은 1000원 단위로 가능합니다."),
+    MINUS("$error 음수는 입력할 수 없습니다."),
+    ZERO("$error 0은 입력할 수 없습니다.")
+}
+
+enum class PRIZE(val prize: Int) {
```

Comment:
```suggestion
enum class PRIZE(
    val prize: Int,
    val apply: (Int) -> String
)
```

이런식으로 매개변수 안에 함수 넣어서 정의하는 방식도 추천합니다.